avm/res/api-management/service/main.bicep

```diff
@@ -130,42 +130,70 @@ param subscriptions array = []
 
 var authorizationServerList = !empty(authorizationServers) ? authorizationServers.secureList : []
 
-var formattedUserAssignedIdentities = reduce(map((managedIdentities.?userAssignedResourceIds ?? []), (id) => { '${id}': {} }), {}, (cur, next) => union(cur, next)) // Converts the flat array to an object like { '${id1}': {}, '${id2}': {} }
-
-var identity = !empty(managedIdentities) ? {
-  type: (managedIdentities.?systemAssigned ?? false) ? (!empty(managedIdentities.?userAssignedResourceIds ?? {}) ? 'SystemAssigned,UserAssigned' : 'SystemAssigned') : (!empty(managedIdentities.?userAssignedResourceIds ?? {}) ? 'UserAssigned' : 'None')
-  userAssignedIdentities: !empty(formattedUserAssignedIdentities) ? formattedUserAssignedIdentities : null
-} : null
+var formattedUserAssignedIdentities = reduce(
```

Comment:
I'd kindly ask to merge #1481 first, to simplify the review of this PR and correctly track the modules still missing v2 formatting

Reply:
done